### PR TITLE
CASMNET-1534 - manual backport of CASMNET-1534-csm-1.2 to main

### PR DESCRIPTION
**Pull request by dle-hpe**
# Description

Backport of CASMNET-1534-csm-1.2 for main
# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/network/management_network/bican_support_matrix.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+* Node Management Network (NMN) \- Provides the internal control plane for systems management and jobs control.
+* Hardware Management Network (HMN) \- Provides internal access to system baseboard management controllers (BMC/iLO) and other lower-level hardware access.
+
+### External and edge networks
+
+* Customer Management Network (CMN) \- Provides customer access from the site to the system for administrators.
+* Customer Access Network (CAN) or Customer High Speed Network (CHN) provide:
+  * Customer access from the site to the system for job control and jobs data movement.
+  * Access from the system to the site for network services like DNS, LDAP, etc.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_support_matrix.md:45]:
```suggestion
  * Access from the system to the site for network services like DNS, LDAP, and so on.
```

File: operations/network/management_network/bican_technical_details.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+        2. [When configuration occurs](#422-when-configuration-occurs)
+        3. [Ability to change post-install](#423-ability-to-change-post-install)
+
+## 1 CAN new features overview
+
+Bifurcation or splitting of the Customer Access Network (CAN) enables customization of customer traffic to and from the system.
+Customization will be performed during installation.
+For CSM 1.2 there are two new customer access networks being introduced as part of the process to split the existing monolithic CAN:
+
+1. **High Speed Customer Access - CHN (Customer High Speed Network)** : This feature adds the ability to connect to User Application Nodes (UAN), UAI, Compute Nodes
```
Comment (nrockershousen) [inline operations/network/management_network/bican_technical_details.md:40]:
```suggestion
1. **High Speed Customer Access - CHN (Customer High Speed Network)** : This feature adds the ability to connect to User Application Nodes (UAN), User Access Instance (UAI), and Compute nodes
```

File: operations/network/management_network/bican_support_matrix.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+### Option A: CMN + CAN (Management Network only - Layer 2 separation)
+
+![cmn plus can](img/cmn_plus_can.png)
+
+### Option B: CMN + CHN (Administration over Management Network, User Access over High Speed Network)
+
+![cmn plus chn](img/cmn_plus_chn.png)
+
+Note: During installation, the High Speed Network is not configured until relatively late in the process.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_support_matrix.md:57]:
```suggestion
Note: During installation, the High Speed Network (HSN) is not configured until relatively late in the process.
```

File: operations/network/management_network/bican_support_matrix.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+Installation generally requires site access for deployment artifacts, site DNS, etc.
+In order to achieve this, the CMN is used during the installation process for system traffic egress until the High Speed Network is available.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_support_matrix.md:59]:
```suggestion
In order to achieve this, the CMN is used during the installation process for system traffic egress until the HSN is available.
```

File: operations/network/management_network/bican_support_matrix.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+#### CMN IPv4 sizing and requirements
+
+* Site routable
+* Contiguous (CIDR block)
+* Non-overlapping with internal networks (configurable during installation)
+* Size estimate is the sum of:
+  * Number of Non-Compute Nodes (NCNs) of type master, worker, or storage used by the Kubernetes cluster
+  * Number of switches on the Management Network
+  * Number of administrative API endpoints
+  * Several administrative addresses for switch interfaces and routing.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_support_matrix.md:89]:
```suggestion
  * Several administrative addresses for switch interfaces and routing
```

File: operations/network/management_network/bican_support_matrix.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+  * A `/26` block is typically sufficient for systems less than approximately 4000 nodes.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_support_matrix.md:90]:
```suggestion
  * A `/26` block is typically sufficient for systems with less than approximately 4000 nodes
```

File: operations/network/management_network/bican_support_matrix.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+### CAN or CHN sizing and requirements
+
+#### CAN or CHN IPv4 sizing and requirements
+
+* Site routable
+* Contiguous (CIDR block)
+* Non-overlapping with internal networks (configurable during installation)
+* Size estimate is the sum of:
+  * Number of application nodes requiring access from the site (User Access Node (UAN), login nodes, etc.)
```
Comment (nrockershousen) [inline operations/network/management_network/bican_support_matrix.md:100]:
```suggestion
  * Number of application nodes requiring access from the site (User Access Node (UAN), login nodes, and more)
```

File: operations/network/management_network/bican_support_matrix.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+  * Number of User Access Instances (UAI) in Kubernetes (if used).
```
Comment (nrockershousen) [inline operations/network/management_network/bican_support_matrix.md:101]:
```suggestion
  * Number of User Access Instances (UAI) in Kubernetes (if used)
```

File: operations/network/management_network/bican_support_matrix.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+  * Number of API endpoints
+  * Several administrative addresses for switch interfaces and routing
+  * NOTE: CAN or CHN sizing is largely dependent on customer-specific use cases and application node hardware.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_support_matrix.md:104]:
```suggestion
  * CAN or CHN sizing is largely dependent on customer-specific use cases and application node hardware
```

File: operations/network/management_network/bican_technical_details.md [hunk 10]
```diff
@@ -0,0 +1,249 @@
...
+
+- External administrator access to administrative endpoints moves to the Customer Management Network (CMN).
+- External user access to user endpoints moves to the Customer High-speed Network (CHN).
+- The CHN is now the default ingress network.
+- Customer access over the legacy CAN is disabled by default.
+
+Reverting or changing any decisions will be manual.
+
+For the feature matrix of the CAN, see [BICAN Support Matrix](bican_support_matrix.md).
+Details of the High Speed CAN (CHN) and the Management CAN (CMN) are described below.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_technical_details.md:59]: The spelled out versions of CHN and CMN vary a across these files. We should keep this consistent.

File: operations/network/management_network/bican_technical_details.md [hunk 11]
```diff
@@ -0,0 +1,249 @@
...
+
+![Customer Access Overview](img/customer_access_overview.png)
+
+## 2 Customer High Speed Network (CHN)
+
+Access to system resources from the customer site over the High Speed Network is provided by the High Speed Customer Access Network (CHN).
+As can be seen in the diagram above, traffic ingress from the site for the CHN is over the edge routers.
+Typically these are a pair of Arista switches which provide other HSN access -- for ClusterStor, for example.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_technical_details.md:67]:
```suggestion
Typically, these are a pair of Arista switches which provide other HSN access -- for ClusterStor, for example.
```

File: operations/network/management_network/bican_technical_details.md [hunk 12]
```diff
@@ -0,0 +1,249 @@
...
+Reverting or changing any decisions will be manual.
+
+For the feature matrix of the CAN, see [BICAN Support Matrix](bican_support_matrix.md).
+Details of the High Speed CAN (CHN) and the Management CAN (CMN) are described below.
+
+![Customer Access Overview](img/customer_access_overview.png)
+
+## 2 Customer High Speed Network (CHN)
+
+Access to system resources from the customer site over the High Speed Network is provided by the High Speed Customer Access Network (CHN).
```
Comment (nrockershousen) [inline operations/network/management_network/bican_technical_details.md:65]:
```suggestion
Access to system resources from the customer site over the High Speed Network (HSN) is provided by the High Speed Customer Access Network (CHN).
```

File: operations/network/management_network/bican_technical_details.md [hunk 13]
```diff
@@ -0,0 +1,249 @@
...
+- NCN images to support additional **subnets and routing.**
+- CFS configurations for CN and UAN **addressing and routing.**
+- UAI to support changes to **addressing and routing.**
+- MetalLB to create new API endpoints and peer with edge router.
+- Arista switch pair to create a new BGP routing instance, and add or modify existing virtual routing instance for path and access control.
+- HSN required for transport of application traffic, so new procedures need to be developed for troubleshooting and support.
+
+#### 2.4.2 When addressing occurs
+
+- Installation, as part of CSI data.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_technical_details.md:137]:
```suggestion
Addressing occurs during installation as part of CSI data.
```

File: operations/network/management_network/bican_technical_details.md [hunk 14]
```diff
@@ -0,0 +1,249 @@
...
+
+#### 2.4.3 Ability to change post-install
+
+- No
```
Comment (nrockershousen) [inline operations/network/management_network/bican_technical_details.md:141]:
```suggestion
Not supported.
```

File: operations/network/management_network/bican_technical_details.md [hunk 15]
```diff
@@ -0,0 +1,249 @@
...
+- Internal to the system, CHN traffic will exist in the same Layer 2 domain with internal HSN traffic until the Slingshot network supports VLAN separation.
+- Compute Node (CN) and Application Node (UAN, in this case) configuration or IPv4 addressing and routing is via CFS.
+  - When multiple HSN interfaces exist, the CHN will be configured on the HSN0 NIC.
+- UAI addressing and routing over the HSN interfaces for the worker NCNs is required.
+- API endpoints in MetalLB for the CHN will be accessible over worker NCN HSN interfaces (via ECMP Layer 3 routing).
+  - MetalLB will peer with the edge routers to supply load balanced API access.
+
+#### 2.5.2 When configuration occurs
+
+- Installation, as part of a virtual routing instance on the edge routers.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_technical_details.md:160]:
```suggestion
Installation, as part of a virtual routing instance on the edge routers.
```

File: operations/network/management_network/bican_technical_details.md [hunk 16]
```diff
@@ -0,0 +1,249 @@
...
+
+#### 3.4.2 When configuration occurs
+
+- During installation, as part of CSI data generation.
+- During installation, as part of Management Network configuration.
+- During installation, as part of NCN deployment.
+
+#### 3.4.3 Ability to change post-install
+
+- Not recommended, and would be manual.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_technical_details.md:223]:
```suggestion
This is not recommended and is a manual process.
```

File: operations/network/management_network/bican_technical_details.md [hunk 17]
```diff
@@ -0,0 +1,249 @@
...
+
+## 4 Customer Access - external/site access (site DNS, LDAP, etc.)
+
+System access to site or external resources (such as the Internet, site DNS servers, and site LDAP servers) was previously provided over the CAN.
+By default this CAN access path remains, but for the CSM 1.2 release it is possible during installation to select system-to-site access over the CHN or CMN.
+
+### 4.1 Traffic separation and routing
+
+At installation time one of the following egress routes from the system to the site must be selected: CAN (default), CHN, CMN.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_technical_details.md:232]:
```suggestion
At installation time one of the following egress routes from the system to the site must be selected: CAN (default), CHN, or CMN.
```

File: operations/network/management_network/bican_technical_details.md [hunk 18]
```diff
@@ -0,0 +1,249 @@
...
+#### 4.2.1 Touchpoints: effects and changes
+
+- Installer customization changes:
+  - Management Network changes possibly for routing, but new ACLs may be necessary.
+  - NCNs will require specific site routes to prioritize selected path over the system default (CAN).
+- Dependent on CHN and CMN work.
+
+#### 4.2.2 When configuration occurs
+
+- During installation, as part of site configuration.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_technical_details.md:245]:
```suggestion
Configuration occurs during installation as part of the site configuration.
```

File: operations/network/management_network/bican_technical_details.md [hunk 19]
```diff
@@ -0,0 +1,249 @@
...
+
+#### 4.2.3 Ability to change post-install
+
+- No
```
Comment (nrockershousen) [inline operations/network/management_network/bican_technical_details.md:249]:
```suggestion
Not supported.
```

File: operations/network/management_network/bican_technical_details.md (hunk 10 above)
Comment (dle-hpe) [reply operations/network/management_network/bican_technical_details.md]: did a pass at making CHN and CMN more consistent

File: install/README.md [hunk 20]
```diff
@@ -16,6 +16,13 @@ like the checking and updating of firmware on system components or the preparati
 
 Once the CSM installation has completed, other product streams for the HPE Cray EX system can be installed.
 
+A major feature of CSM 1.2 is the Bifurcated CAN (BICAN).
+The BICAN is designed to separate administrative network traffic from user network traffic.
+More information can be found on the [BICAN summary page](../operations/network/management_network/bican_technical_summary.md).
+Review the BICAN summary before continuing with the CSM 1.2 install.
```
Comment (mharding-hpe) [inline install/README.md:22]:
```suggestion
Review the BICAN summary before continuing with the CSM install.
```
```suggestion
Review the BICAN summary before continuing with the CSM 1.2 install.
```

File: operations/network/management_network/bican_technical_summary.md [hunk 21]
```diff
@@ -0,0 +1,24 @@
...
+- CMN – Customer Management Network
+- CHN – Customer High Speed Network
+- NMN – Node Management Network
+- HMN – Hardware Management Network
+
+## BICAN features
+
+- Bifurcation or splitting of the Customer Access Network (CAN) enables customization of customer traffic to and from the system.
+Customization will be performed during installation.
+In CSM 1.2, as part of the process to split the existing monolithic CAN, two new customer access networks are introduced:
```
Comment (mharding-hpe) [inline operations/network/management_network/bican_technical_summary.md:20]:
```suggestion
In CSM 1.2, as part of the process to split the existing monolithic CAN, two new customer access networks were introduced:
```

File: operations/network/management_network/bican_technical_summary.md [hunk 22]
```diff
@@ -0,0 +1,24 @@
...
+
+## BICAN features
+
+- Bifurcation or splitting of the Customer Access Network (CAN) enables customization of customer traffic to and from the system.
+Customization will be performed during installation.
+In CSM 1.2, as part of the process to split the existing monolithic CAN, two new customer access networks were introduced:
+  - High Speed Customer Access - CHN: This feature adds the ability to connect to Application Nodes (UAN), UAI, Compute Nodes,
+and Kubernetes API endpoints from the customer site via the High Speed Network (HSN).
+  - Management Customer Access - CMN:  Using a new VLAN on the Management Network, this feature allows system administrative access from the customer site.
+Administrative access was previously available on the original CAN; this feature provides a traffic path and access split.
```
Comment (mharding-hpe) [inline operations/network/management_network/bican_technical_summary.md:24]:
```suggestion
Prior to CSM 1.2, administrative access was previously available on the original CAN; this feature provided a traffic path and access split.
```

File: install/README.md [hunk 23]
```diff
@@ -16,6 +16,13 @@ like the checking and updating of firmware on system components or the preparati
 
 Once the CSM installation has completed, other product streams for the HPE Cray EX system can be installed.
 
+A major feature of CSM 1.2 is the Bifurcated CAN (BICAN).
```
Comment (mharding-hpe) [inline install/README.md:19]:
```suggestion
A major feature of CSM is the Bifurcated CAN (BICAN).
```

**Review by mharding-hpe — APPROVED** [review]

**Pull request merged**